triton/nonblocking_stream.py

```diff
@@ -10,6 +10,8 @@
 Adapted from https://github.com/rhettg/BlueOx/blob/master/blueox/network.py
 
 """
+from __future__ import unicode_literals
+from builtins import str
```

Comment:
This overrides the _normal_ built-in `str` with a `str` function that behaves like Python3's `str` function (in python3, since all strings are unicode, there is no `unicode` function). I mainly did this import because of the use of `str` [here](https://github.com/postmates/triton-python/blob/joe-astr/unicode-patch-for-struct-in-nonblocking/triton/nonblocking_stream.py#L77), but @Syeoryn just pointed out how this could cause problems, so I'm removing it.